src/components/certificate-viewer/certificate-viewer.tsx

```diff
@@ -12,20 +12,53 @@ import { Certificate, TExtension, EnumOIDs } from '../../utils/crypto';
 })
 export class CertificateViewer {
   cert: Certificate;
+  interaction: Element;
 
   /**
    * The certificate value for decode and show details. Use PEM or DER format.
    */
   @Prop() certificate: string;
 
   componentWillLoad() {
+    this.interaction = undefined;
+
+    if (!this.certificate) {
```

Comment:
Bad decision. I think we don't need the new property to render the state. Need to use `cert `:
1. `cert` is empty - show the empty state.
2. `cert` has error - show the error state.